centreon/src/Centreon/Infrastructure/Contact/ContactRepositoryRDB.php

```diff
@@ -202,7 +205,9 @@ public function findByAuthenticationToken(string $token): ?Contact
                     ON t.topology_page = COALESCE(contact.default_page, template.default_page)
                 INNER JOIN `:db`.security_authentication_tokens sat
                     ON sat.user_id = contact.contact_id
-                WHERE sat.token = :token
+                INNER JOIN `centreon`.security_token st
```

Comment:
Shouldn't we use the placeholder for dbnames ?
```suggestion
                INNER JOIN `:db`.security_token st
```